front/lib/plans/trial.ts

```diff
@@ -0,0 +1,17 @@
+import type { SubscriptionType } from "@dust-tt/types";
+
+import type { Subscription } from "@app/lib/models";
+import type { PlanAttributes } from "@app/lib/plans/free_plans";
+import { FREE_TEST_PLAN_DATA } from "@app/lib/plans/free_plans";
+import { TRIAL_PLAN_CODE } from "@app/lib/plans/plan_codes";
+
+export const TRIAL_PLAN_DATA: PlanAttributes = {
+  ...FREE_TEST_PLAN_DATA,
```

Comment:
Should be on top of PRO_PLAN_SEAT_29_CODE no ?

Reply:
I also think this can live in pro_plans, no need for this file?

Reply:
Yeah it makes sense, let me move it.

Reply:
Wait actually the pro plan is only stored in DB, we don't have any JS representation of it.

Reply:
@spolu is it ok if I'm adding one more call to fetch the pro plan, in the `Authenticator` or I keep hardcoding the values to save this extra call?

Reply:
(Discussed on slack)